src/injectors/github-injector.ts

```diff
@@ -15,6 +15,18 @@ namespace Gitpodify {
     export const CSS_REF_NO_CONTAINER = "no-container";
 }
 
+function isNewNavigation(): boolean {
+    if (select.exists(".file-navigation")) {
+        return false;
+    }
+    const newNavigationFileHelperElement = 'button[data-hotkey="t,T"][hidden]';
+    if (select.exists(newNavigationFileHelperElement)) {
+        return true;
+    }
+
+    return false;
+}
```

Comment:
This is the important bit, it's hacky, but works for now. For the transitioning phase it should work and we can adjust.